src/resources/elements/primeDesignSystem/ptoggle/ptoggle.scss

```diff
@@ -72,15 +72,15 @@
     +.pToggleSlider {
       cursor: not-allowed;
       background: transparent;
-      border: 1px solid $Neutral02;
+      border: 1px solid $Secondary05;
```

Comment:
disabled colors should be purple (not gray)

Reply:
Should revert this, was a mistake in the figma.  Should be Neutral02